packages/storybook/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "lib",
+    "target": "es6",
+    "module": "es6",
+    "lib": ["dom", "es6"],
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "types": ["react"],
```

Comment:
I still don't think this should be here. Also if you're not using webpack-env it should be removed from dependencies.

Reply:
If you do remove `react` and don't add `webpack-env`, it's important to use an empty array instead to prevent automatic inclusion of all `@types` in the compilation, which is very slow.

Reply:
Why do you need `react`? Even without this the types will be automatically picked up in any file where you import `react`. The only time you need to reference it like this is if you want to pull React into the ambient global namespace.

Reply:
yes i also just realized the same. fixed!